.github/workflows/ci.yaml

```diff
@@ -111,7 +111,7 @@ jobs:
 
       # Check the lockfile
       - name: Validate lockfile updates
-        run: cargo update --locked
+        run: cargo check --locked
```

Comment:
[We have check in line 92](https://github.com/OpenZeppelin/openzeppelin-relayer/pull/161/files#diff-944291df2c9c06359d37cc8833d182d705c9e8c3108e7cfe132d61a06e9133ddR92)

Reply:
this one is specifically to verify if there are any minor/patch updates to the dependencies and fail the step if it's not updated. This will be automated going forward to automatically commit to the PR's if the lockfile is failed.